src/pages/ReimbursementAccount/BusinessInfo/substeps/WebsiteBusiness.tsx

```diff
@@ -58,6 +59,10 @@ function WebsiteBusiness({reimbursementAccount, user, session, onNext, isEditing
         shouldSaveDraft: isEditing,
     });
 
+    useEffect(() => {
+        BankAccounts.addBusinessWebsiteForDraft(defaultCompanyWebsite);
+    }, [defaultCompanyWebsite]);
```

Comment:
@NikkiWines hey sorry, I forgot to add back this logic in https://github.com/Expensify/App/pull/48520 which I mentioned in the plan here.

Reply:
Oops, I forgot to put the link. Yes, that's the one.